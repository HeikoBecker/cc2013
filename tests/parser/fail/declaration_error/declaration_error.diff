1c1,3
< declaration_error.c:1:6: error: Expected ';' but found 'EOF'
---
> declaration_error.c:1:5: error: FunctionDefinition has to be either (void) or all parameters have to be declared
> int i
>     ^^^^
