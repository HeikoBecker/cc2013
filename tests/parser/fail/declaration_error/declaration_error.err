declaration_error.c:1:5: error: FunctionDefinition has to be either (void) or all parameters have to be declared
int i
    ^^^^
1 error(s)
