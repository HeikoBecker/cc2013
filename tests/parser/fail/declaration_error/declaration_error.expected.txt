declaration_error.c:1:6: error: Expected ';' but found 'EOF'
1 error(s)
